aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/PinpointClientFactory.java

```diff
@@ -47,7 +46,7 @@
     private PinpointClientFactory() {
     }
 
-    static AnalyticsClient create(Context context,
+    static PinpointManager create(Context context,
```

Comment:
The class that creates a `PinpointManager` should probably be called a `PinpointManagerFactory`, not a `PinpointClientFactory`, I guess.